glom/core.py

```diff
@@ -215,6 +224,11 @@ def _unpack_stack(scope):
         cur, nxt = stack[i], stack[i + 1]
         if cur[3] == nxt[3]:
             cur[3] = None
+    if only_errors:  # trim the stack to the last error
+        # leave at least 1 to not break formatting func below
+        # TODO: make format_target_spec_trace() tolerate an "empty" stack cleanly
```

Comment:
when does an empty stack happen, and what's the behavior here?

Reply:
the stack formatting is a bit of an ad-hoc mess right now; I think this may have been something unrelated that I just ran across